src/main/scala/sbtprotobuf/ProtobufPlugin.scala

```diff
@@ -90,7 +90,10 @@ class ScopedProtobufPlugin(configuration: Configuration, private[sbtprotobuf] va
     },
     protobufExcludeFilters ++= {
       val dirs = sourceDirectories.value
-      dirs.map(d => Glob(d.toPath()) / "google" / "protobuf" / "*.proto")
+      dirs.flatMap(d => Seq(
+        Glob(d.toPath()) / "google" / "protobuf" / "*.proto",
+        Glob(d.toPath()) / "com" / "google" / "protobuf" / "*.proto",
```

Comment:
This is required because new protobuf-java contains proto file under `com/google/protobuf` in addition to `google/protobuf`.